[2023-06-01T11:42:13.450+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2023-06-01T11:35:43.326175+00:00 [queued]>
[2023-06-01T11:42:13.466+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2023-06-01T11:35:43.326175+00:00 [queued]>
[2023-06-01T11:42:13.468+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:42:13.469+0000] {taskinstance.py:1284} INFO - Starting attempt 3 of 8
[2023-06-01T11:42:13.470+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:42:13.486+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-06-01 11:35:43.326175+00:00
[2023-06-01T11:42:13.493+0000] {standard_task_runner.py:55} INFO - Started process 3784 to run task
[2023-06-01T11:42:13.499+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'manual__2023-06-01T11:35:43.326175+00:00', '--job-id', '948', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpfua6xmtd']
[2023-06-01T11:42:13.501+0000] {standard_task_runner.py:83} INFO - Job 948: Subtask insert_into_table
[2023-06-01T11:42:13.589+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table manual__2023-06-01T11:35:43.326175+00:00 [running]> on host 687ba427e841
[2023-06-01T11:42:13.668+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2023-06-01T11:35:43.326175+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-01T11:35:43.326175+00:00
[2023-06-01T11:42:13.670+0000] {sql.py:253} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-06-01', 'dag_with_postgres_operator_v03')
        
[2023-06-01T11:42:13.681+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:42:13.971+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:42:13.985+0000] {sql.py:364} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-06-01', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-06-01T11:42:13.989+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-01T11:42:14.004+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20230601T113543, start_date=20230601T114213, end_date=20230601T114214
[2023-06-01T11:42:14.033+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:42:14.059+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
